spring-web-modules/spring-mvc-basics-4/src/main/webapp/WEB-INF/index.jsp

```diff
@@ -1,5 +0,0 @@
-<html>
-<body>
```

Comment:
in which folder we are moving these templates, they may not be needed in the WEB-INF with boot-2 but they could still be needed and should be moved to the directory accessed by thymeleaf

Reply:
This file is not referred anywhere, that is why I removed some of the unused files.